contracts/pallet-ibc/src/lib.rs

```diff
@@ -304,6 +311,15 @@ pub mod pallet {
 	pub type ChannelsConnection<T: Config> =
 		StorageMap<_, Blake2_128Concat, Vec<u8>, Vec<(Vec<u8>, Vec<u8>)>, ValueQuery>;
 
+	#[pallet::storage]
+	#[allow(clippy::disallowed_types)]
+	pub type WhitelistChannelIds<T: Config> =
```

Comment:
we need to add extrinsic to add channe ids. can you add a comment of its structure. why does it map to ()?

Reply:
Done.